Sources/Hypertext.swift

```diff
@@ -49,8 +49,9 @@ extension Array: Renderable {
 }
 
 open class tag: Renderable {
-    public var name: String? = nil
-    public var isSelfClosing: Bool = false
+    open var isSelfClosing: Bool { return false }
+    open var name: String { return String(describing: type(of: self)) }
```

Comment:
I can see that setting a `String` for `name` can be extra work, but I think that it lets us be explicit and then also flexible with what we name our subclasses. And for the initial set of tags that are in `Tags.swift`, we as the maintainers carry that initial effort and name all of them :)